crates/fpe/python/cloudproof_fpe/__init__.pyi

```diff
@@ -0,0 +1,133 @@
+from typing import List, Optional, Tuple, Union
+
+class Alphabet:
+    """
+    The `Alphabet` class brings Format Preserving Encryption (FPE) functions
+    like encryption and decryption on a parametrize alphabet
+    """
+
+    def __init__(self, alphabet_id: str):
+        """
+        Initialize the FormatPreservingEncryption instance with a given
+        alphabet type.
+
+        :param alphabet_type: A string indicating the type of alphabet to be used
+        for the encryption. Currently, only the following options are supported:
+        'alpha', 'alpha_lower', 'alpha_numeric', 'alpha_numeric_sym', 'latin1',
+        'latin1_sup', 'numeric', 'hex', and 'base64'.
```

Comment:
not consistent with   Numeric = "numeric",
  Hexadecimal = "hexa_decimal",
  AlphaLower = "alpha_lower",
  AlphaUpper = "alpha_upper",
  Alpha = "alpha",
  AlphaNumeric = "alpha_numeric",
  Utf = "utf",
  Chinese = "chinese",
  Latin1Sup = "latin1sup",
  Latin1SupAlphanum = "latin1sup_alphanum",

Reply:
thanks